cmd/server-mux.go

```diff
@@ -457,6 +460,8 @@ func (m *ServerMux) ListenAndServe(certFile, keyFile string) (err error) {
 			// Execute registered handlers, update currentReqs to keep
 			// tracks of current requests currently processed by the server
 			atomic.AddInt32(&m.currentReqs, 1)
+			lmsg := fmt.Sprintf("serving for %s", r.URL.String())
+			log.logger.Info(lmsg)
```

Comment:
Please ping us on https://slack.minio.io  or me `@y4m4b4` , we will help you out further if you have any questions. Feel free to close this PR.